src/sentry/static/sentry/app/components/version.jsx

```diff
@@ -17,20 +15,16 @@ var Version = React.createClass({
   },
 
   render() {
-    var version = this.props.version;
+    var version = encodeURIComponent(this.props.version);
```

Comment:
Not sure if this is necessary ... was the intention to escape this (e.g. to mitigate xss)? And if so, why not org slug / project slug.